manual/src/html_processing/Makefile

```diff
@@ -1,3 +1,5 @@
+include ../../../Makefile.config
```

Comment:
The compiler Makefile style for those definition is to define first a `ROOTDIR` directory:
```Makefile
ROOTDIR = ../../..
include $(ROOTDIR)/Makefile.common
```